src/commands.jl

```diff
@@ -179,6 +181,25 @@ function next_line!(@nospecialize(recurse), frame::Frame, istoplevel::Bool=false
 end
 next_line!(frame::Frame, istoplevel::Bool=false) = next_line!(finish_and_return!, frame, istoplevel)
 
+"""
+    pc = until!(recurse, frame, line=nothing istoplevel=false)
```

Comment:
`until_line!`, here and below (GitHub won't let me make suggestions when I'm reviewing only one of the 2 commits here)